virttest/qemu_vm.py

```diff
@@ -4315,6 +4315,20 @@ def check_block_locked(self, value):
                     return block['locked']
         return False
 
+    def get_block_virtual_size(self, dev_dict={}):
```

Comment:
please move this function to qemu_monitor.py

Reply:
Thanks for your advice~

Reply:
Hi xutian, 

I moved this function on qemu_monitor.pyhttps://github.com/avocado-framework/avocado-vt/pull/1465/, but some colleagues confused this change.
For this patch, I have some explanation:
1. I written in qemu_vm.py before due to there is a get_block_dev function, I think get_block_virtual_size is similar with get_block_dev; 
2. I written in qemu_monitor.py, look on functional, I'm not so clear for you considers, I just see this function used monitor.info("block") directly.
3. I asked for xuhan, he think it's a function related storage device, it's better to write on  block related files, like storage.py. What do you think for you?

Thanks~

Reply:
Hi xutian,
Also discussed with xuhan, he think should write in a block related file, so create a new block.py file on this patch  https://github.com/avocado-framework/avocado-vt/pull/1490, later any block related functions or classes can write in block.py.

Best Rgards~